pkg/devcontainer/config/parse.go

```diff
@@ -67,6 +67,46 @@ func SaveDevContainerJSON(config *DevContainerConfig) error {
 	return nil
 }
 
+func ParseDevContainerJSONFile(jsonFilePath string) (*DevContainerConfig, error) {
```

Comment:
nit: docblock for a public function please explaining what is does :pray:

Reply:
Done